Command/Message/example2.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
naming of the example is incorrect. please see chapter 2 of [contribution manual](https://bit.ly/contribution_manual).

Reply:
Changed the name of the examples to "example-***.json"